sources/osgShadow/shaders/shadowReceive.glsl

```diff
@@ -170,11 +187,24 @@ float shadowReceive(const in bool lighted,
 
         // Now computes Shadow
 #ifdef _OUT_DISTANCE
-        float res = getShadowPCF(shadowTexture, shadowSize, shadowUV, shadowReceiverZ, shadowBiasPCF, clampDimension, outDistance);
-        if (lighted) shadow = res; 
+        float res = getShadowPCF(shadowTexture,
+                                 shadowSize,
+                                 shadowUV,
+                                 shadowReceiverZ,
+                                 shadowBiasPCF,
+                                 clampDimension,
+                                 outDistance
```

Comment:
see comments above OPT_ARG_INSTANCE_outDistance to simplify a bit a reduce the #define branching code to after the getShadowPCF call